engine/chain/internal/bhs/heathcheck.go

```diff
@@ -0,0 +1,34 @@
+package bhs
+
+import (
+	"context"
+	"time"
+
+	"github.com/bitcoin-sv/go-paymail/spv"
+	"github.com/bitcoin-sv/spv-wallet/engine/chain/errors"
+	"github.com/bitcoin-sv/spv-wallet/engine/chain/models"
+	"github.com/bitcoin-sv/spv-wallet/engine/spverrors"
+)
+
+var pastMerkleRoot = []*spv.MerkleRootConfirmationRequestItem{{
+	// This is a known merkle root from the blockchain
+	// timestamp: 2024-10-04 10:55:37 UTC
+	MerkleRoot:  "f67ae53720205a55f4e99c632debabb68b6df0dc0f68affd200a076aee6e80e6",
+	BlockHeight: 864921,
+}}
+
+// HealthcheckBHS checks the health of the Block Headers Service (BHS).
+// It verifies the BHS is reachable and ready to verify merkle roots by providing a known merkle root.
+func (s *Service) HealthcheckBHS(ctx context.Context) error {
```

Comment:
Could this healthcheck be a simple GET to `/status` endpoint in BHS? I think they would actually check the same if the server response, and neither of them checks if the server is responsive i.e 1 second after the 1st request failed as mentioned in one of my PRs. It would simplify logic a little but maybe I'm missing something. WDYT?

Reply:
Nope, I guess 🙈 
Please check out how it is done right now in `check_block_header_service.go`.

We need to check if `BHS` can verify our transactions - that's why we need to have relatively updated/synced working instance of BHS - and this way we can check that. 

Maybe the name `HealthcheckBHS` is missleading, but:
- I wanted to reflect what we already have in  `check_block_header_service.go`
- I recall we discussed about it a few months ago and decided to do it that way. 

But maybe we want to change that logic 🤔 WDYT @dorzepowski @wregulski

Reply:
I don't remember this discussion, but if we already discussed this, and agreed upon this solution, then lets keep it as is.

Reply:
So let's keep it 👌